docs/python/reference/configuration.md

```diff
@@ -24,29 +24,34 @@ class DBOSConfig(TypedDict):
     name: str
     database_url: Optional[str]
     sys_db_name: Optional[str]
-    app_db_pool_size: Optional[int]
     sys_db_pool_size: Optional[int]
     db_engine_kwargs: Optional[Dict[str, Any]]
     log_level: Optional[str]
     otlp_traces_endpoints: Optional[List[str]]
-    run_admin_server: Optional[bool]
+    otlp_logs_endpoints: Optional[List[str]]
     admin_port: Optional[int]
+    run_admin_server: Optional[bool]
 ```
 
 - **name**: Your application's name.
-- **database_url**: A connection string to a Postgres database. The supported format is
-```
-postgresql://[username]:[password]@[hostname]:[port]/[database name]
-```
-The `sslmode=require`, `sslrootcert` and `connect_timeout` parameter keywords are also supported.
-Defaults to:
+- **database_url**: A connection string to a Postgres database. DBOS will use the connection string, unmodified, to create a [SQLAlchemy engine](https://docs.sqlalchemy.org/en/20/core/engines.html).
+
+:::info
+Passwords in connection strings should be escaped (e.g., with [urllib](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote)).
+:::
+
+:::info
+DBOS Transact requires the [psycopg 3](https://www.psycopg.org/psycopg3/docs/) driver and will accordingly set the driver to `postgresql+psycopg` before creating an engine. If you use [alembic](https://alembic.sqlalchemy.org/en/latest/) to manage database migrations, you'll need to set the driver in your connection string.
```

Comment:
Why does the second sentence need to be there?  Of COURSE if you use Alembic you need to set a driver, what we we changing?

Reply:
I found a somewhat un-intuitive in our current behavior: if you set your `DBOS_DATABASE_URL` w/o the driver, it'll fail for the migration but work otherwise.

I thought about making the driver mandatory in the user-provided connection string. But such a connection string won't work with other tools like `psql`.